unitTests/ui/MissingTermManagerTest.ts

```diff
@@ -64,6 +65,24 @@ export function MissingTermsManagerTest() {
         element.click();
         expect(missingTerms.length).toBe(1);
       });
+
+      it('should log an event when it is clicked', () => {
```

Comment:
I think the description would be clearer if you mentioned the type of event. `...log a removeMissingTerm event...`